src/shared/WizardPage/Form.jsx

```diff
@@ -115,12 +115,14 @@ export class WizardFormPage extends Component {
         )}
         <form className={className}>{children}</form>
         <div className="grid-row" style={{ marginTop: '0.5rem' }}>
-          <div className="grid-col-12 text-right margin-top-6 tablet:margin-top-3">
+          <div className="grid-col margin-top-6 tablet:margin-top-3">
             {!isMobile && (
               <button className="usa-button usa-button--outline cancel" onClick={this.cancelFlow}>
```

Comment:
This button still kinda floats - what about something like this:

```suggestion
               <button className="usa-button usa-button--outline cancel padding-left-0" onClick={this.cancelFlow}>
```

Which produces this result:

<img width="1031" alt="Screen Shot 2019-11-15 at 10 08 03" src="https://user-images.githubusercontent.com/3522323/68957527-d7c5be80-078f-11ea-83ee-8c96bfd138ae.png">

Reply:
yea, that works for me. Just changed it.